msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -11,10 +11,15 @@ ms.topic: overview
 You can allow orgs (specifically Teams admins) to customize some aspects of your Microsoft Teams app. Some possible examples include:
 
 * Changing the app's accent color to match the org's brand.
-* Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see. (Note: Users adding a connector to a chat will still see the original app name, *Contoso*).
+* Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see. 
+   > [!NOTE] 
+   > Users adding a connector to a chat will still see the original app name, *Contoso*.
 
 You can enable this feature in the [Developer Portal for Teams](https://dev.teams.microsoft.com/home). (This configures `configurableProperties` in your app manifest.) For more information, see [customize apps in Microsoft Teams](/MicrosoftTeams/customize-apps).
 
+> [!NOTE]
+> App customization is not supported for sideloaded and LOB apps. You cannot test the feature by sideloading the app.
```

Comment:
Does this mean this feature is only available for apps published to the Teams store? If so, we should probably state that instead. We could still keep the second sentence about sideload testing.

Reply:
I agree to Heath's comment here, also instead of adding it as a note, can we mention this as Testing instructions? 
`Testing Instructions: 
App customization is not supported for sideloaded and LOB apps. You cannot test the feature by sideloading the app. To test this feature, submit the new app manifest and use a test tenant to verify your changes`

Reply:
Edit as per suggestion.